src/Domain/FileProcessors/SniffFileProcessor.php

```diff
@@ -59,7 +66,15 @@ public function processFile(SplFileInfo $splFileInfo): void
         $file = $this->fileFactory->createFromFileInfo($splFileInfo);
         $file->processWithTokenListenersAndFileInfo(
             $this->tokenListeners,
-            $splFileInfo
+            $splFileInfo,
+            $this->fixEnabled
         );
+
+        if ($this->fixEnabled === true && $file->getFixableCount() !== 0) {
+            $file->activeFix();
+            $file->fixer->fixFile();
+            file_put_contents($splFileInfo->getPathname(), $file->fixer->getContents());
```

Comment:
This is the line that actually replaces to code with the fixed code?

Reply:
Yes :wink: